src/CollapsedCellOptimizer.cpp

```diff
@@ -449,6 +664,18 @@ void optimizeCell(std::vector<std::string>& trueBarcodes,
         std::exit(74);
       }
 
+      if ( dumpCellEq ){
+        std::vector<std::vector<uint32_t>> labelsEq ;
+        std::vector<uint32_t> countsEq ;
+        labelsEq.resize(salmonEqclasses.size());
+        countsEq.resize(salmonEqclasses.size());
+        for(size_t salEqId = 0 ; salEqId < salmonEqclasses.size(); ++salEqId){
+          labelsEq[salEqId] = salmonEqclasses[salEqId].labels;
+          countsEq[salEqId] = salmonEqclasses[salEqId].count;
+        }
+        gzw.writeDedupCellEQVec(trueBarcodeIdx, labelsEq, countsEq, true);
+      }
+
       if ( numBootstraps and noEM ) {
```

Comment:
Add numGibbs here, just cross check if we have any other cases like this.
Another thing to check here or in alevinUtils would be that both gibbs and bootstaps are not provided simultaneously to the command line

Reply:
This particular place is for dumping gene-level equivalence classes. Nothing to do with Gibbs. Sorry if I missed what you meant.

Reply:
Huh ? I meant line 679. Basically checka if noEM is provided along with numBootstraps it gives error. We might wanna add numGibbs here and check similar such cases if exist.

Reply:
Gotcha! done